src/System.Private.CoreLib/shared/System/Diagnostics/StackTrace.cs

```diff
@@ -299,7 +303,8 @@ internal string ToString(TraceFormat traceFormat)
                         // Append original method name e.g. +MoveNext()
                         sb.Append('+');
                         sb.Append(methodName);
-                        sb.Append('(').Append(')');
+                        sb.Append('(');
+                        sb.Append(')');
```

Comment:
I haven't tested... is `sb.Append('('); sb.Append(')');` significantly faster than `sb.Append("()");`, and if so, is it in the context of this method?  Otherwise, I'd just assume using the simpler call.